src/main/kotlin/de/bund/digitalservice/useid/model/IdentityAttributes.kt

```diff
@@ -0,0 +1,6 @@
+package de.bund.digitalservice.useid.model
+
+data class IdentityAttributes(
+    val DG1: String,
```

Comment:
It's probably enough for the first draft, but there are some more: https://github.com/digitalservicebund/useid-eservice-example/blob/main/useid-eservice-sdk/src/DataGroup.ts

Reply:
I can add them.